pinecone/sparse/bm25/bm25-quora.ipynb

```diff
@@ -0,0 +1,2451 @@
+{
```

Comment:
<blockquote>For this, we need a <a href="https://app.pinecone.io/" target="_blank">free API key</a>. We initialize the connection like so: </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>